dash/development/_r_components_generation.py

```diff
@@ -322,6 +334,24 @@ def generate_js_metadata(pkg_data, project_shortname):
     return function_string
 
 
+# determine whether dependency uses async or dynamic flag
+# then return the properly formatted string if so, i.e.
+# " async = TRUE,". a dependency can have async or
+# dynamic elements, neither of these, but never both.
+def check_async_type(dep):
```

Comment:
Would suggest renaming to `get_**` instead as this doesn't check but rather returns some specific value to use as-is

Reply:
fixed in 3ff7607bec58bc128d4367669d31f121cc49bbb4